packages/angular-cli/lib/config/schema.json

```diff
@@ -61,6 +61,12 @@
           "test": {
             "type": "string"
           },
+          "coverageExcludes": {
```

Comment:
For example, appRoot/testing/test.helper.ts.  I understand as a workaround I could've used test.helper.spec.ts but that was too messy IMO.

I'm going to check out the karma-remap-istanbul 0.3.0 fix that @felipesilva suggested when I get home later.  The only difference there being no glob support, but that could be implemented on the karma-remap-istanbul side if really necessary (and it probably isn't).

So at any rate I'll check out that solution and probably recommend closing this in favor of updating to ^0.3.0 karma-remap-istanbul and adding documentation to guide people to that solution.

If you have strong feelings one way or another let me know, otherwise thanks to both of you for the input!

I only wonder why I couldn't find @felipesilva's solution when I looked!  Oh well, this was a good and fun learning experience regardless!